{{cookiecutter.app_name}}/.eslintrc.json

```diff
@@ -11,10 +11,17 @@
   },
   "extends": [
     "eslint:recommended",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended"{% if cookiecutter.use_openapi == "yes" %},
+    "@nimblehq/eslint-config-nimble",
+    "plugin:yml/recommended"{% endif %}
   ],
   "globals": {
     "Atomics": "readonly",
     "SharedArrayBuffer": "readonly"
   }
-}
+}{% endif %}{% if cookiecutter._both_variant == "no" and cookiecutter.use_openapi == "yes" %}{
```

Comment:
Is this for the web or API only and with open API? Shouldn't it's for the API only alone because the web one already have it above 🤔

Reply:
@carryall This logic is for the API-only variant. If the developer choose `web only` variant, we will not support openapi as there are no APIs